www/js/pages/sign-in/components/oauth/oauth.controller.js

```diff
@@ -45,12 +45,16 @@
 
         $scope.$on("authorize", (e, response) => {
             if (response) {
+                const accountList = response.account_list;
                 accounts.forEach((value, index) => {
-                    if (index > 0) {
```

Comment:
You shouldn't remove the if statement because by removing it, the first account in `accounts` will be added two times in localStorage, since the authorized account has been added to localStorage in `sign-in.controller.js` between line 65 to 78.